src/test/java/seedu/address/ui/PersonCardTest.java

```diff
@@ -21,14 +21,20 @@ public class PersonCardTest {
 
```

Comment:
All tests pass when I run this test file individually but fails when run with the other tests